src/inputs/ObjectSearch.tsx

```diff
@@ -130,9 +130,12 @@ class ObjectSearch extends Component<IObjectSearchProps> {
     this.search = value;
     this.isLoading.setTrue();
     this.updateValueCaches();
-    const response = await getEndpoint(`/${endpoint}/${toKey(params)}`);
-    this.options = response.results;
-    this.isLoading.setFalse();
+    try {
+      const response = await getEndpoint(`${endpoint}${toKey(params)}`);
+      this.options = get(response, 'results', []);
+    } finally {
+      this.isLoading.setFalse();
```

Comment:
No errors are raised here if search fails?

Reply:
Ah good point, no! Hmmm... how _should_ we handle that? Obviously, something like 5xx should maybe display a notification but... unsure about things like 4xx.

Reply:
Maybe just display it if its 5xx?

Reply:
I started to implement toast notifications for errors, but upon thinking about it: In FormManager we make sure never to send toast notifications the library user can't stop or override. I think this would probably be a bad place to break that rule. But you're right, silently catching and squashing it is bad too. So instead I added a console.error, what do you think about that?

Reply:
I think that's probably the best solution for now!